deploy/charts/Chart.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v2
+name: openebs-ndm
+description: Kubernetes Storage Device Management. Also used to maintain the inventory of block devices.
+version: 0.8.1
+appVersion: 0.8.1
+icon: https://raw.githubusercontent.com/cncf/artwork/master/projects/openebs/icon/color/openebs-icon-color.png
+home: http://www.openebs.io/
+keywords:
+  - cloud-native-storage
+  - block-storage
+  - ndm
+  - disk-inventory
+  - storage
+sources:
+  - https://github.com/openebs/node-disk-manager
+maintainers:
+  - name: akhilerm
+    email: akhil.mohan@mayadata.io
+  - name: kmova
```

Comment:
@sonasingh46 - can you replace my name with Michael ( @xUnholy )

Reply:
@xUnholy -- Can you please comment your email address here that I can put in here?

Reply:
This email should work with notifications 20387402+xUnholy@users.noreply.github.com

Unless @kmova you'd rather my contactable email?

Reply:
Done